paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -715,18 +715,55 @@ internal class PaymentSheetViewModelTest {
         assertThat(isEnabled).isTrue()
 
         viewModel.savedStateHandle[SAVE_PROCESSING] = true
+
+        idleLooper()
```

Comment:
How come all of these idleLoopers are needed now? This seems like a behavior change.